cli/require-protobufjs.js

```diff
@@ -0,0 +1,19 @@
+"use strict";
+
+module.exports = function requireProtobufjs() {
+    try {
+        // local development, i.e. forked from github
+        require.resolve("..");
```

Comment:
Can you educate me why do you need these two no-op calls to `require.resolve()`? My understanding is that they would've made sense to check if the module is available, but in both cases the regular `require` follows - and will fail if there is no module found.

Reply:
(is the behavior of `require.resolve` different from the one of `require`? I was under impression that `require` would throw the same `Error` if the requested module is not found)

Reply:
you are correct, require.resolve is unnecessary. I removed those calls now